src/server/system_services/schemas/bucket_schema.js

```diff
@@ -139,6 +140,9 @@ module.exports = {
             type: 'string',
             enum: ['DISABLED', 'SUSPENDED', 'ENABLED']
         },
+        creation_date: {
+            type: 'string',
+        },
```

Comment:
We don't store dates as strings in the DB, look at how we do it for other DB schemas.
Also this kind of change would require an upgrade script to work for existing buckets.
However I think we can skip the DB schema for this purpose since we already have a date for every entity that can be extracted from `bucket._id.getTimestamp().getTime()`,  so you could just add it to the bucket api and make bucket_server reply with that date from the _id.

Reply:
Nice, Thanks!
I was wondering how we could get the create time on a bucket that already exists.
This is the reason we did not add an upgrade script.

Reply:
Updated and ready for review now. Thanks